File: TP/TP1/Code/5E.cpp
/* Afficher une ligne de longueur N, avec deux étoiles, et N - 2 espaces */
#include <iostream>

int main()
{
  int n;
  
  std::cout << "Quelle est la longueur de la ligne ?" << std::endl;
  std::cin >> n;
  
  for (int i = 0; i < n; ++i) {
    if (i == 0 || i == n - 1)
      std::cout << "*" << std::flush;
    else
      std::cout << " " << std::flush;
  }
  
  std::cout << std::endl;
  return 0;
}

File: TP/TP1/Code/Python/5E.py
#!/usr/bin/python
# Afficher une ligne de longueur N, avec deux étoiles, et N - 2 espaces
print('Donne moi une valeur : ', end='')
foo = int(input())

for i in range(foo):
    if i == 0 or i == foo - 1:
        print('*', end='')
    else:
        print(' ', end='')
